api/handler/space.go

```diff
@@ -0,0 +1,121 @@
+package handler
+
+import (
+	"fmt"
+	"log/slog"
+	"net/http"
+	"slices"
+
+	"github.com/gin-gonic/gin"
+	"opencsg.com/csghub-server/api/httpbase"
+	"opencsg.com/csghub-server/common/config"
+	"opencsg.com/csghub-server/common/types"
+	"opencsg.com/csghub-server/common/utils/common"
+	"opencsg.com/csghub-server/component"
+)
+
+func NewSpaceHandler(config *config.Config) (*SpaceHandler, error) {
+	tc, err := component.NewSpaceComponent(config)
```

Comment:
`sc` would be a better name for the `SpaceComponent`.

Reply:
updated